misk/src/main/kotlin/misk/MiskTestingServiceModule.kt

```diff
@@ -18,24 +18,51 @@ import com.google.inject.spi.InstanceBinding
 import com.google.inject.spi.LinkedKeyBinding
 import com.google.inject.spi.ProviderInstanceBinding
 import com.google.inject.util.Types
+import misk.environment.FakeEnvVarModule
+import misk.environment.RealEnvVarModule
 import misk.healthchecks.HealthCheck
 import misk.inject.KAbstractModule
 import misk.metrics.MetricsModule
 import misk.moshi.MoshiModule
 import misk.prometheus.PrometheusHistogramRegistryModule
 import misk.resources.ResourceLoaderModule
+import misk.resources.TestingResourceLoaderModule
 import misk.time.ClockModule
 import misk.tokens.TokenGeneratorModule
 import javax.inject.Singleton
 
-class MiskServiceModule : KAbstractModule() {
+/**
+ * [MiskRealServiceModule] should be installed in real environments.
+ */
+class MiskRealServiceModule : KAbstractModule() {
+  override fun configure() {
+    install(ResourceLoaderModule())
+    install(RealEnvVarModule())
+    install(MiskCommonServiceModule())
+  }
+}
+
+/**
+ * [MiskTestingServiceModule] should be installed in unit testing environments.
```

Comment:
does this suggest that there are tests that would use `MiskRealServiceModule`? Can't think of any that we would actually use in practice.

Reply:
perhaps some sort of true end-to-end system test that actually provides a real env.

Reply:
I don't think so. It would be too much a hassle to setup a "real env" on your host machine (unless you were using docker)

Reply:
I added some more javadoc to help explain this should only fake out things about the env, not include application layer fakes